schema.sql

```diff
@@ -0,0 +1,18 @@
+create or replace table wp_wporg_gp_translation_events_actions
+(
+    event_id       int(10)     not null comment 'ID of the event',
+    user_id        int(10)     not null comment 'ID of the user who authored the translation',
+    translation_id int(10)     not null comment 'ID of the translation',
+    created_at     datetime    not null comment 'When the translation was created, in UTC',
```

Comment:
@akirk @amieiro We discussed and agreed on removing `created_at`, but I think we need to keep it. 

Because the start and end datetimes of the event can be edited, it's possible that the table contains entries that no longer belong in the event. For example

1. The event is created with start datetime A
2. The event starts (now = A)
3. User creates a translation
4. The event is edited to start at datetime B (which is later than A)
5. The table would contain a translation that no longer belongs in the event

To fix this, we could prevent the event start and end datetimes from being edited after the event has started. However, I think that would be poor UX for a user who creates/edits events.

If, for example, the user made a mistake and only realises it after the event started, they would probably create a new event, so you'd now have two entries for the same event, which would be confusing. So the user would also probably want to to delete events.

Instead, we could remove all these problems by having stats calculation filter out actions that took place in a datetime that does not fit the current start and end datetimes of the event.

To be able to do this, the `created_at` column needs to exist, I think.

This turned out to be a longer text than I originally thought.

Does this all make sense? I think it's important to ensure sure stats are correct.

Reply:
I would prefer to just use `insert ignore` instead of `replace` so that this cannot happen.